content/events/hacktoberfest.adoc

```diff
@@ -14,56 +14,52 @@ opengraph:
-You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
-Generally, any pull requests in GitHub would qualify.
-See the link:/participate/[Contribute and Participate] page for more information about how to contribute.
+During this event everyone can support open-source by contributing changes, and then earn a unique digital reward.
+This year marks the 10th anniversary of Hacktoberfest.
 
+We invite everyone to participate and to contribute to the Jenkins project, regardless of their background and Jenkins experience.
+There are many ways to link:/participate/[contribute] to Jenkins during Hacktoberfest.
+You can work on code or documentation, contribute to link:/doc/developer/internationalization/, or create new link:/artwork[artwork], etc.
+You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
+Generally, any four non-spammy pull requests in GitHub or merge requests in GitLab would qualify.
+Please see the link:/participate/[Contribute and Participate] page for more information about how to contribute.
 
 == Quick start
 
 1. Sign-up to Hacktoberfest on link:https://hacktoberfest.com[the event website].
-2. Join link:https://app.gitter.im/#/room/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
-3. Everything is set, just start creating pull-requests!
-** If a repository has no `hacktoberfest` topic set, contact the maintainers to see if they are willing to accept a contribution.
-
+2. Join link:https://matrix.to/#/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
+3. Everything is set, just start creating pull or merge requests!
+** If a repository has no `hacktoberfest` topic is set, contact the maintainers to see if they are willing to accept a contribution.
+** Alternatively, if a pull or merge request has the label `hacktoberfest-accepted`, it will count towards your total.
+** Your pull or merge request must not be labeled as invalid to be counted.
+** For your pull or merge requests to be accepted, your pull or merge requests must be merged, or have the `hacktoberfest-accepted` label, or have an overall approving review on GitHub.
 
 == Where to contribute?
 
-The Jenkins project is spread across several organizations on GitHub (`link:https://github.com/jenkinsci/[jenkinsci]`, `link:https://github.com/jenkins-infra/[jenkins-infra]`, `link:https://github.com/jenkins-zh/[jenkins-zh]`).
+The Jenkins project is spread across several organizations on GitHub (`link:https://github.com/jenkinsci/[jenkinsci]` and `link:https://github.com/jenkins-infra/[jenkins-infra]`.).
 You are welcome to contribute to **any** repository in **any** of those organizations, or any other Jenkins-related repository on GitHub.
 Repositories may have different contribution guidelines, review and merge policies.
```

Comment:
```suggestion
The Jenkins project is spread across two organizations on GitHub: `link:https://github.com/jenkinsci/[jenkinsci]` and `link:https://github.com/jenkins-infra/[jenkins-infra]`.
You are welcome to contribute to *any* repository in *either* of those organizations or any other Jenkins-related repository on GitHub.
Be aware that repositories may have different contribution guidelines, review processes, and merge policies.
```

I'm updating the text since there are only two organizations now listed here, and adjusting some of the other text.